app/src/main/assets/chart.html

```diff
@@ -48,7 +48,7 @@
       </tr>
       {% for row in rows %}
         {% set id = row.item.conceptIds | first %}
-        <tr class="obs concept-{{id}}">
+        <tr class="obs concept-{{id}} {{ row.item.label | tosafechars }}">
```

Comment:
This css class is only to help espresso find the values based on the label and not on numbers. This facilitates the development and reading of tests.  We should never use this classes for styling.